package.json

```diff
@@ -72,12 +74,10 @@
   },
   "lint-staged": {
     "**/*.ts?(x)": [
-      "npm run lint:fix",
-      "git add"
+      "npm run lint:fix"
     ],
     "**/*.{scss,css,html}": [
-      "prettier --print-width 120 --write",
-      "git add"
+      "prettier --print-width 120 --write"
```

Comment:
Because they were not formatted beforehand.
It's easy to fall into the trap of `--no-verify`... and then scrupulous fella will pay the price later on.